db_scripts/mosip_regprc/dml/regprc-transaction_type.csv

```diff
@@ -8,16 +8,13 @@ BIO,transaction_done,eng,TRUE,MOSIP_SYSTEM,now()
 PACKET_RECEIVER,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 UPLOAD_PACKET,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 VALIDATE_PACKET,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
-VIRUS_SCAN,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 OSI_VALIDATE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 EXTERNAL_INTEGRATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 DEMOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 BIOGRAPHIC_VERIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 UIN_GENERATOR,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
 NOTIFICATION,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
-PRINT_SERVICE,transaction done,eng,TRUE,MOSIP_SYSTEM,now()
```

Comment:
PRINT_SERVICE is currently being used,I have requested for removal of PRINT

Reply:
added PRINT_SERVICE